torchx/Makefile

```diff
@@ -9,6 +9,9 @@ PRIV_DIR = $(MIX_APP_PATH)/priv
 LIBTORCH_LIB_DIR = $(PRIV_DIR)/lib
 TORCHX_SO = $(PRIV_DIR)/torchx.so
 
+CMAKE_BUILD_DIR ?= $(shell pwd)/_build/cmake
```

Comment:
should we build to `cache` instead of `_build`? Or if we build to `_build`, should it be the mix-env-scoped directory? `_build/dev/torchx/cmake` or something like that?